src/ext.js

```diff
@@ -155,33 +173,84 @@ export default function ext({ translator, shouldHide, senseNavigation }) {
-                  show: (data) => checkShowAction(data, 'automation'),
+                  show: (data) => checkShowAction(data, 'automation') && !multiUserAutomation,
+                },
+                automationId: {
+                  type: 'string',
+                  component: 'expression-with-dropdown',
+                  translation: 'Object.ActionButton.Automation',
+                  ref: 'automationId',
+                  dropdownOnly: false,
+                  options: async () => getAutomations(),
+                  show: (data) => checkShowAction(data, 'automation') && multiUserAutomation,
+                  change: async (data) => {
+                    const a = await getAutomation(data.automationId)
+                    data.automationShowTriggered = a.runMode === 'triggered';
+                    if(data.automationTriggered) {
+                      data.automationExecutionToken = a.executionToken
+                    }
+                    else {
+                      data.automationExecutionToken = ''
+                    }
+                  },
                 },
-                // Boolean property to instruct the automation action to create a
-                // bookmark and send it to the selected automation in the
-                // property panel.
                 automationPostData: {
                   type: 'boolean',
                   ref: 'automationPostData',
                   translation: 'Object.ActionButton.Automation.SendSelections',
                   show: (data) => checkShowAction(data, 'automation'),
                   defaultValue: false,
                 },
+                automationShowTriggered: {
+                  type: 'boolean',
+                  ref: 'automationShowTriggered',
+                  show: () => false,
+                  default: false,
+                },
+                automationTriggered: {
+                  type: 'boolean',
+                  ref: 'automationTriggered',
+                  translation: 'Object.ActionButton.Automation.RunModeTriggered',
+                  show: (data) => checkShowAction(data, 'automation') && multiUserAutomation && data.automationShowTriggered,
+                  defaultValue: false,
+                  change: async (data) => {
+                    const a = await getAutomation(data.automationId)
+                    if(data.automationTriggered) {
+                      data.automationExecutionToken = a.executionToken;
+                    }
+                    else {
+                      data.automationExecutionToken = '';
+                    }
+                  },
+                },
+                automationTriggeredText: {
+                  label: `Object.ActionButton.Automation.RunModeTriggeredHelp`,
+                  component: 'text',
+                  show: (data) => checkShowAction(data, 'automation') && multiUserAutomation && data.automationShowTriggered,
+                },
+                automationExecutionToken: {
```

Comment:
same here, should not be in this file if show is always false

Reply:
done in the latest commit

Reply:
Have you not pushed that commit?

Reply:
sorry I missed this and thought it was there - should be there now in the latest commit

Reply:
Which commit?

Reply:
this one here: https://github.com/qlik-oss/sn-action-button/pull/332/commits/508467210f40b98108faf14005658f8a77990e70

Reply:
Should not `automationExecutionToken` also be removed as it's never shown?